_data/navigation.yml

```diff
@@ -532,6 +532,9 @@ primary:
           - text: Mural (collaborative whiteboard tool)
             url: tools/mural/
             internal: true
+          - text: Qualtrics (survey management tool)
+            url: qualtrics/
```

Comment:
```suggestion
            url: software-tools/qualtrics/
```

Reply:
I think I gave bad advice before, thinking all of software tools were in a `tools` directory. That's what I get for not looking. Anyways, I think the above should work.